breathecode/admissions/serializers.py

```diff
@@ -447,7 +447,10 @@ def validate(self, data):
                 'That user already exists in this cohort')
 
         if ('role' in request_item and request_item['role'] == 'TEACHER' and
```

Comment:
instead of `== TEACHER` lets validate `!= 'STUDENT'` because assistants and reviewers have to also be staff memebers.